src/components/ThreadEnvelope.vue

```diff
@@ -431,8 +433,16 @@ export default {
 	.left {
 		flex-grow: 1;
 	}
-	.source-modal ::v-deep .modal-container {
-		overflow-y: scroll !important;
+	.source-modal {
+		::v-deep .modal-container {
+			height: 800px;
```

Comment:
how responsive is this? Would that work on mobile, e.g. in landscape mode?

Reply:
It works fine on small screens. I tested it by resizing the browser window to small sizes.

The Modal component (from nextcloud/vue) applies `max-height: 80%` to the modal container.